Mage.Sets/src/mage/cards/a/AbaddonTheDespoiler.java

```diff
@@ -31,13 +26,13 @@
  */
 public final class AbaddonTheDespoiler extends CardImpl {
 
-    private static final FilterStackObject filter = new FilterSpell();
+    private static final FilterSpell baseFilter = new FilterSpell();
 
     static {
-        filter.add(new CastFromZonePredicate(Zone.HAND));
-        filter.add(AbaddonTheDespoilerPredicate.instance);
+        baseFilter.add(new CastFromZonePredicate(Zone.HAND));
     }
 
+    //TODO: this hint isn't used. Use it or clean it up
```

Comment:
I believe you should add `this.addAbility(markOfTheChaosAscendant.addHint(hint));` after line 62 (`this.addAbility(markOfTheChaosAscendant);`).

Search for `final Hint` in the project to see some examples of how to use it.